charts/thin/templates/fuse/daemonset.yaml

```diff
@@ -118,23 +124,12 @@ spec:
         - name: thin-conf
           secret:
             secretName: {{ template "thin.fullname" . }}-fuse-conf
-            defaultMode: 0755
+            defaultMode: 0444
+        - name: runtime
+          configMap:
+            name: {{ template "thin.fullname" . }}-runtime-info
```

Comment:
Typo here: configMap name should be like `{name}-runtimeset`

Reply:
thanks!

Reply:
thanks!